docs/citation.rst

```diff
@@ -0,0 +1,23 @@
+Citation
+________
+
+VAPOR is developed as an Open Source application by the National Center for Atmospheric Research, under the sponsorship of the National Science Foundation. Continued support for VAPOR is de
+pendent on demonstrable evidence of the software's value to the scientific community. You are free to use VAPOR as permitted under the terms and conditions of the license. We kindly request, however, that you cite VAPOR in your publications and presentations. We suggest the following citations as appropriate:
+
+For journal articles, proceedings, etc.,
+we request:
+
+**Visualization & Analysis Systems Technologies. (2022) Visualization and Analysis Platform for Ocean, Atmosphere, and Solar Researchers (VAPOR version 3.6.1) [Software].  Boulder, CO: UCAR/NCAR - Computational and Information System Lab.  doi:10.5065/d6j38qhc**
```

Comment:
This references a specific version, 3.6.1. How does this information get updated, manually or automagically?

Reply:
The version will need to be updated manually with each release, as well as the DOI if we're going to retrieve new ones with Zenodo.

Reply:
I think you can have this automated. See [here](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content)

Otherwise, let's be sure to get it on our release checklist :-)

Reply:
It appears that Zenodo [can issue an all-versions-DOI](https://github.com/zenodo/zenodo/issues/1582) that redirects to the latest release.  I do not see the option for us yet, and I believe it's because we haven't issued a "Latest" release since enabling Zenodo.

I'll open a ticket to revisit this after we release, at which point we should have an all-versions-DOI that can be used where we provide citation information.